### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by ungoldman**
- update `.travis.yml` template with more recent node versions
- update `CHANGELOG.md` template
- add scripts for generating a gh-pages site

Hi @Flet! Mind taking a quick look to make sure I'm not doing anything too crazy?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 language: node_js
 node_js:
-  - 'stable'
-  - '0.12'
   - '0.10'
-sudo: false
+  - '0.12'
```
Comment (Flet) [inline .travis.yml]: Probably OK at this point to drop < v4? Probably even drop v5... :)

File: templates/.travis.yml.mustache [hunk 2]
```diff
@@ -1,9 +1,10 @@
 language: node_js
 node_js:
-  - 'stable'
-  - '0.12'
   - '0.10'
-sudo: false
+  - '0.12'
```
Comment (Flet) [inline templates/.travis.yml.mustache]: ...same here...

**Comment by Flet** [comment]
Oops, looks like I just missed it! :)

**Comment by ungoldman** [comment]
No worries, realized it's really nothing major. Thanks for taking a look! You're right, probably fine to drop 0.10, 0.12, & 5 from travis and the template too.

**Comment by ungoldman** [comment]
It is done: 9bdfacab7106fe18a7af9102030f98e3f6e03c6c

**Comment by Flet** [comment]
Sweet! Will this package live on or will init-module eventually take over as your preferred package kickstarter?

**Comment by ungoldman** [comment]
¯\\\_(ツ)\_/¯

Would be nice to update `module-init` but I don't use it as often as I used to. `init-module` could be more useful too. Both need a little work. You have any ideas/thoughts/preferences?

**Comment by Flet** [comment]
I really like `module-init` and I use it to start just about all new repos (even if I end up removing some things if its an internal project). Its helped me not forget about keeping changelogs and filling out READMEs :)

There isn't anything critical I can think of at the moment that its missing. It might be nice to have a way to spit out a nice browserify/webpack build/config setup and maybe a simple express server.js with dependencies sorted, but even that feels like it could be too much.

**Comment by ungoldman** [comment]
Thanks, that's nice to hear 😁 

I think `init-module` could be made more interesting if it had more npm config options related to what `module-init` does and a post-init hook to generate files (not sure how to go about doing that part). Right now the only thing it's really useful for is as a `package.json` editor using the `npm init` interface. Maybe one day it will be a worthy successor to `module-init` but for now I think this is getting the job done alright.

**Pull request merged**